### DISCUSSION TITLE
package org.springframework.data.jpa.domain.support does not exist

### DISCUSSION THREAD

**Discussion by ajaykushwaha1** [quarkusio/quarkus#22019] [Community]
Hi, 

I am migrating my project from spring-boot to quarkus and using the below dependency for spring-data-jpa.

**groupId: io.quarkus
    artifactId: quarkus-spring-data-jpa
     version: 2.5.0.Final**

I have to use AuditingEntityListener for auditing purpose. But I am getting compilation error :  **package org.springframework.data.jpa.domain.support does not exist.** Can any one help on this?

I am using below import for AuditingEntityListener,

 **org.springframework.data.jpa.domain.support.AuditingEntityListener**

![image](https://user-images.githubusercontent.com/36269701/145198895-ec4c153c-d758-4ec7-8604-6909ae1ca8bf.png)

**Comment by maxandersen** [comment]
@geoand  @Sanne  ^^ 

I don't think we have direct support for this. 

If AuditingEntityListener is a pure JPA compliant listener you should be able to just add its dependency just not guaranteed that will work in native mode.

**Reply by Sanne** [reply]
@maxandersen all JPA listeners work fine now both in JVM and native.

see #20708

**Comment by geoand** [comment]
We already have https://github.com/quarkusio/quarkus/issues/22020 for this...

**Reply by gastaldi** [reply]
I think we can delete this discussion since it's already being tracked in #22020?

**Reply by maxandersen** [reply]
lets not delete discussions - just lock them and point to where tracked.

**Reply by gastaldi** [reply]
Ah, didn't know we could lock discussions, yeah, that makes sense
